dashboard/pages/dependency_graph.tsx

```diff
@@ -90,11 +244,38 @@ export default function StreamingGraph() {
           alignItems="flex-start"
           flexDirection="column"
         >
-          <Text fontWeight="semibold" mb={3}>
-            Relations
-          </Text>
           <Box flex={1} overflowY="scroll">
             <VStack width={SIDEBAR_WIDTH} align="start" spacing={1}>
+              {/* NOTE(kwannoel): No need to reset prometheus bp, because it is stateless */}
+              <Button onClick={(_) => toggleResetEmbeddedBackPressures()}>
+                Reset Back Pressures
```

Comment:
It might be worth adding some explanation (hover or footnote) about why it needs reset. Not very easy to understand to users.

Reply:
Yeah separate PR. This PR should just be for relation dependency graph. Anyway this is a minor convenience. If you click away to other tabs and click back, it can also reset 😆